src/components/Transaction.vue

```diff
@@ -27,10 +27,20 @@
       <template slot="header">Transfer warning</template>
       <div>
         <div class="_padding-bottom-1">
-          You are about to transfer money to an address that doesn't have a zkSync balance yet. The transfer will happen inside zkSync L2. If you want to move money from zkSync to
-          the mainnet, please use the
-          <nuxt-link :to="`/withdraw?w=${inputtedAddress}`">Withdraw</nuxt-link>
-          function instead.
+          You are about to transfer money to an address that does not have a zkSync balance yet. Please be aware of the common pitfalls and loss of funds!
```

Comment:
Remove first list element (lines: 30-39) and place lines 40-41 instead of the text we've had before